packages/gatsby-theme-newrelic/src/components/GlobalHeader.js

```diff
@@ -537,10 +400,62 @@ const GlobalHeader = ({ className, activeSite, hideSearch = false }) => {
   );
 };
 
+export const NR_SITES = {
```

Comment:
Can we move this to the constants file at `packages/gatsby-theme-newrelic/src/utils/constants.js`?